packages/docs/blog/2021-07-09-remotion-2-2.md

```diff
@@ -0,0 +1,165 @@
+---
+slug: 2-2
+title: Remotion 2.2
+author: Jonny Burger
+author_title: Remotion Creator
+author_url: https://github.com/JonnyBurger
+author_image_url: https://avatars2.githubusercontent.com/u/1629785?s=460&u=12eb94da6070d00fc924761ce06e3a428d01b7e9&v=4
+---
+
+import { ThreeDPlayer } from "../components/3DPhonePlayer.tsx";
+
+Welcome to the release notes of Remotion 2.2! It's been a while since v2.1, but in the meanwhile we had a dozen smaller releases inbetween. This post summarizes the highlights of the past 2 months 🎉.
+
+## Environment variable support
+
+Us developers use environment variables extensively to manage configuration, secrets and other stuff you don't want to commit to a Github repo. If you write videos with code, environment variables are useful too!
+
+We added support for environment variables from the CLI, using a `.env` file, and allowing you to pass variables via our Node.JS APIs too. [Click here to learn more.](/docs/env-variables)
+
+## `<Freeze />` component
+
+This new core component will freeze all of it's children and make them think that the video is paused at a certain time.
+
+Use it for example to display a still frame from a video:
+
+```tsx twoslash title="MyVideo.tsx"
+import {Freeze, Video} from 'remotion';
+import myVid from './vid.mp4';
+
+export const MyVideo = () => {
+  return (
+    <Freeze frame={30}>
+      <Video src={myVid} />
+    </Freeze>
+  )
+}
+```
+
+However, it works for any content. As you can see the API is dead simple. You can combine it with the [`<Sequence>`](/docs/sequence) API to make any content play, pause, and then continue again.
+
+[Read more about the `<Freeze />` component here.](/docs/freeze)
+
+## Video + Audio `playbackRate`
+
+This prop allows you to slow down or speed up video and audio elements! Now you can import a video in normal speed and play it in slow motion or timelapse without rerendering the video.
+
+```tsx twoslash title="LightningSpeed.tsx"
+import {Video} from 'remotion';
+import myVid from './vid.mp4';
+
+export const MyVideo = () => {
+  return (
+    <Video src={myVid} playbackRate={4}/>
+  )
+}
+```
+
+While previewing, we are using the HTML5 `playbackRate` API, and when rendering, we will calculate the correct FFMPEG command for any tempo.
+
+Learn more about the `playbackRate` props for [`<Audio />`](/docs/audio/#controlling-playback-speed) and [`<Video />`](/docs/video/#controlling-playback-speed).
+
+## `@remotion/three` package
+
+In case you missed it, we released a new helper package for [React Three Fiber](https://github.com/pmndrs/react-three-fiber)!
+
+<ThreeDPlayer />
+<br/>
+
+Three.JS is admittedly not easy, but with the work that [Poimandres](https://github.com/pmndrs) is doing in making it more approachable and more integrated with React, this is changing rapidly. Creating 3D videos in React and rendering them to real MP4 videos, we are making it possible!
```

Comment:
Three.JS is admittedly not easy, but with the work that [Poimanders] is doing *to make* it more approachable and more integrated with React, this is changing rapidly.